enhancements/network/upgrades-with-ovn-ic.md

```diff
@@ -101,6 +101,9 @@ data:
   zone-mode: singlezone
   temporary: true
 ```
+
+The "temporary: true" field above is only used up until 4.14.13 and is omitted in later 4.14.z.
```

Comment:
hmm should we stretch this a bit further and make it "omitted in releases >= 4.14.14 and >= 4.15 ?" I don't know if talking specifically about 4.14 z stream implies that 4.15 and other minor versions are not even effected by this..same comment here and elsewhere you mention versioning..
or maybe in context of upgrades 4.13 -> 4.15 will always go via 4.14 so we are ok here?

Reply:
It only ever applies to upgrades to 4.14.z. There's no skipping releases when you upgrade :)